lib/id3c/cli/command/etl/__init__.py

```diff
@@ -159,6 +160,14 @@ def upsert_encounter(db: DatabaseSession,
 
         LOG.info(f"Updating existing encounter {encounter.id}")
 
+        new_row_hash = db.fetch_row("""
+            select md5(row ( %(individual_id)s::integer, %(site_id)s::integer, %(encountered)s::timestamp with time zone, %(age)s::interval, %(details)s::jsonb)::text)
+        """, data)
```

Comment:
Is the casting here necessary here but not in the initial select because of a difference in how execute and fetch_row are interacting with the data types?

Reply:
The casting here is to ensure that the incoming values are the same data type and format as the stored values prior to hashing. For example the incoming `encountered` value may be "2022-03-16" but its stored as "2022-03-16 00:00:00-07"